### ISSUE TITLE
training yolov8 pose-precision=0 recall=0

### ISSUE THREAD

**Issue by s0966066980** [ultralytics/ultralytics#13017]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Train

### Bug

![2024-05-22 18-37-19 的螢幕擷圖](https://github.com/ultralytics/ultralytics/assets/45198905/44fd00ac-b1e8-4e48-8983-826109b05a57)

我透過yolo pose train data=coco8-pose.yaml model=yolov8n-pose.yaml pretrained=yolov8n-pose.pt epochs=100 imgsz=640
得到的結果

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
Hello! Thanks for reaching out and providing detailed information about the issue you're encountering with training the YOLOv8 pose model.

From your description, it seems like the model is not learning as indicated by the zero precision and recall. Here are a couple of things you might want to check:

1. **Data Integrity**: Ensure that the `coco8-pose.yaml` file correctly points to the training and validation data and that the annotations are properly formatted and accurate.
2. **Model Configuration**: Verify that the `yolov8n-pose.yaml` model configuration matches the requirements for pose estimation tasks. Check for any discrepancies in the model architecture or data preprocessing steps.
3. **Learning Parameters**: Sometimes, training issues can arise from suboptimal learning rates or other hyperparameters. Consider experimenting with different learning rates or using a learning rate scheduler.

If these suggestions don't resolve the issue, it would be helpful to see more details about the loss values during training or any error messages you might be receiving. This information could provide further insights into what might be going wrong.

Feel free to share any updates or additional information, and we'll do our best to help you get this resolved! 🚀
